sdks/java/io/file-based-io-tests/pom.xml

```diff
@@ -294,6 +294,11 @@
                     <version>${apache.hadoop.version}</version>
                     <scope>runtime</scope>
                 </dependency>
+                <dependency>
+                    <groupId>javax.xml.bind</groupId>
+                    <artifactId>jaxb-api</artifactId>
```

Comment:
Please update Gradle file as well.

Reply:
Gradle was updated in #4870 Diff: https://github.com/apache/beam/pull/4870/files#diff-76558bfd90b93d37ab9369862732ffd6 It's already merged to the master branch. 
This jenkins jobs works, because when tests are running by perfkit on jenkins, jenkins is clonning master branch and run tests against that code. Let me know if you still prefer to add this line in gradle here?